packages/api/src/generated/graphql.ts

```diff
@@ -1017,6 +1025,7 @@ export type Highlight = {
   quote?: Maybe<Scalars['String']>;
   reactions: Array<Reaction>;
   replies: Array<HighlightReply>;
+  representation: RepresentationType;
```

Comment:
I'm starting to wonder if representation is the best word now. It makes sense in the context of "available representations" but do you think there is a better term?

Reply:
could we use `source`?

Reply:
@jacksonh